templates/jail.local.j2

```diff
@@ -1,3 +1,9 @@
+{% if ansible_distribution_major_version | int >= 12 %}
```

Comment:
I guess it'd check only the version but not the distribution (whether it is debian 12)

Reply:
I've added also debian distribution check, see https://github.com/geerlingguy/ansible-role-security/commit/0b9a774f875ff3c175fc4f16a6d2b5b479d17e45 tasks/fail2ban.yml